src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/RolesController.kt

```diff
@@ -66,7 +66,7 @@ class RolesController(
     rolesService.createRole(createRole)
   }
 
-  @PreAuthorize("hasRole('ROLE_ROLES_ADMIN')")
+  @PreAuthorize("hasAnyRole('ROLE_ROLES_ADMIN', 'ROLE_VIEW_ADMINISTRABLE_USER_ROLES')")
   @Operation(
     summary = "Get role details",
     description = "Get role details, role required is ROLE_ROLES_ADMIN",
```

Comment:
does `'ROLE_VIEW_ADMINISTRABLE_USER_ROLES` need adding to the description and security?